source/libs/group-buttons.tsx

```diff
@@ -19,7 +19,7 @@ export const groupButtons = (buttons: Element[]) => {
 	// If it doesn't exist, wrap them in a new group
 	if (!group) {
 		group = <div class="BtnGroup"/>;
-		wrapAll(buttons, group);
+		wrapAll(buttons, group!);
```

Comment:
I don't understand why TS needs help here, since it's clearly defined...

Reply:
Ah that can be removed. It's because it doesn't understand the JSX properly in regards to types. See my note in initial PR comment.

Reply:
Done.